charts/vector/templates/rbac.yaml

```diff
@@ -5,6 +5,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: {{ include "vector.fullname" . }}
+  labels:
+    {{- include "vector.labels" . | nindent 4 }}
```

Comment:
Note to self: https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace 😄

Reply:
Oh yes, so much of helm templating is whitespace control since yaml is space sensitive.